recipes/aubio/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "aubio" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "2acdb92623b9d4ba641c387760ffe3ec1e4c6ab498e64e5e2286c99e36ffbff8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://aubio.org/pub/{{ name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy x.x
+    - ffmpeg
+  run:
+    - python
+    - numpy x.x
+    - ffmpeg
+
+test:
+  imports:
+    - aubio
+
+about:
+  home: https://aubio.org
+  license: GPL-3.0
+  license_family: GPL
+  license_file: COPYING
+  summary: 'tools for audio and music analysis and transformation'
+  description: |
+    aubio provides a set of tools for analysis and transformation of sounds.
+    Numpy arrays are used to store vectors and matrices, while computations
+    are done in C, for speed and memory efficiency.
+  doc_url: https://aubio.org/manual/latest
+  dev_url: https://github.com/aubio/aubio/
+
+extra:
+  recipe-maintainers:
+    - piem
+    - MartinHN
```

Comment:
@MartinHN - please confirm if you are willing to be a co-maintainer for  the `python-aubio` package on conda-forge.